fatiando/seismic/conv.py

```diff
@@ -0,0 +1,161 @@
+"""
+Zero-offset convolutional seismic modeling
+
+Give a depth model and obtain a seismic zero-offset convolutional gather. You 
+can give the wavelet, if you already have, or use one of the existing, from 
+which we advise ricker wavelet (rickerwave function).
+
+* :func:`~fatiando.seismic.conv.seismic_convolutional`: given the depth 
+  velocity model and wavelet, it returns the convolutional seismic gather.
+* :func:`~fatiando.seismic.conv.rickerwave`: calculates a ricker wavelet.
+
+**References**
+
+Yilmaz, Öz,
+Ch.2 Deconvolution. In: YILMAZ, Öz. Seismic Data Analysis: Processing, 
+Inversion, and Interpretation of Seismic Data. Tulsa: Seg, 2001. Cap. 2. 
+p. 159-270. Available at: <http://dx.doi.org/10.1190/1.9781560801580.ch2>
+
+
+----
+
+"""
```

Comment:
TravisCI [is complaining](https://travis-ci.org/fatiando/fatiando/builds/62279201#L670) of a non-ascii character. Probably the "Öz". To get Unicode characters, try putting this comment on the first line of the file:

    # -*- coding: utf-8 -*-